crates/sui-framework/deps/move-stdlib/sources/string.move

```diff
@@ -3,8 +3,9 @@
 
 /// The `string` module defines the `String` type which represents UTF8 encoded strings.
 module std::string {
-    use std::vector;
+    use std::ascii;
```

Comment:
I'm guessing the changes in this file are spurious / a hold over from the other PR?